apps/docs/src/app/core/component-docs/icon/examples/icon-example.component.scss

```diff
@@ -1,24 +1,22 @@
-.grid {
+.fd-docs-icon-container {
     display: flex;
     flex-wrap: wrap;
+    justify-content: center;
 }
 
-.grid > * {
-    width: calc(100% / 6);
-    margin-bottom: 1rem;
-}
-
-.grid * {
-    text-align: center;
-    display: block;
-}
+.fd-docs-icons {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    word-break: break-all;
+    padding: 1rem;
+    width: 9rem;
 
-.grid fd-icon {
-    margin-bottom: 0.25rem;
-}
+    fd-icon {
```

Comment:
can you apply specificity to this? (You can put it inside `.fd-docs-icons` block). The chances are small but it can still leak and cause problems when the icons are used in the docs).

Reply:
I am seeing that it is inside that block

Reply:
yes, @InnaAtanasova, it is inside the block already : 
![image](https://user-images.githubusercontent.com/53509521/119596673-3ea04b00-bdfd-11eb-8ae1-5a7188a216a6.png)

Reply:
I need glasses :D Sorry about that!